sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -662,6 +663,101 @@ class FileStreamSourceSuite extends FileStreamSourceTest {
     }
   }
 
+  testWithUninterruptibleThread("read data from outputs of another streaming query") {
```

Comment:
nit: you can merge the latest master and use `test` directly. Not need to use `testWithUninterruptibleThread` after #16947